doc/manpages/options.asciidoc

```diff
@@ -5,6 +5,37 @@ NAME
 ----
 options - a
 
+Description
+-----------
+
+Options are used for user configuration.
+You can set the builtin ones or declare your owns.
+
+The following commands are available to interact with them:
+
+----------------------------------------------
+declare-option [-hidden] <type> <name> [value]
+set-option [-add] <scope> <name> <value>
+update-option <scope> <name>
+unset-option <scope> <name>
+----------------------------------------------
+
+Refer to the 'scopes' documentation page for more information about
+scopes.
+
+To get the current value of an option:
```

Comment:
This bit duplicates the contents of the `expansions` page, which increases the probability of having discrepancies in the documentation when either page changes.

#1470 did the same thing as well, if pages' topics intersect then we should redirect one to the other with a "for more information about XXX, refer to the YYY page" type message, instead of potentially creating more opportunities for desynchronization.